tools/sce_clone_handler.py

```diff
@@ -0,0 +1,37 @@
+import subprocess
+
+class SceCloneHandler:
+	project_names = {
+		"core-v4": "https://github.com/SCE-Development/Core-v4.git",
+		"sce-discord-bot": "https://github.com/SCE-Development/SCE-discord-bot.git",	
+		"discord-bot": "https://github.com/SCE-Development/SCE-discord-bot.git",
+		"quasar": "https://github.com/SCE-Development/Quasar.git"
+	}
+
+	def __init__(self, projects):
+		if projects == None:
+			self.projects = ["core-v4"]
+		self.projects = projects
```

Comment:
when `projects` is None here, `self.projects` is set to None at the end of the function